tests/ops/service/privacy_request/test_bigquery_enterprise_privacy_request.py

```diff
@@ -0,0 +1,147 @@
+import time
+from datetime import datetime, timezone
+from typing import Any, Dict, List, Set
+from unittest import mock
+from unittest.mock import ANY, Mock, call
+from uuid import uuid4
+
+import pydash
+import pytest
+
+from fides.api.models.audit_log import AuditLog, AuditLogAction
+from fides.api.models.privacy_request import (
+    ActionType,
+    CheckpointActionRequired,
+    ExecutionLog,
+    ExecutionLogStatus,
+    PolicyPreWebhook,
+    PrivacyRequest,
+    PrivacyRequestStatus,
+)
+from fides.api.schemas.masking.masking_configuration import MaskingConfiguration
+from fides.api.schemas.masking.masking_secrets import MaskingSecretCache
+from fides.api.schemas.policy import Rule
+from fides.api.service.masking.strategy.masking_strategy import MaskingStrategy
+from tests.ops.service.privacy_request.test_request_runner_service import (
+    get_privacy_request_results,
+)
+
+PRIVACY_REQUEST_TASK_TIMEOUT = 5
+# External services take much longer to return
+PRIVACY_REQUEST_TASK_TIMEOUT_EXTERNAL = 60
+
+
+@pytest.mark.integration_bigquery
+@pytest.mark.integration_external
+@pytest.mark.parametrize(
+    "dsr_version",
+    ["use_dsr_3_0", "use_dsr_2_0"],
+)
+@mock.patch("fides.api.models.privacy_request.PrivacyRequest.trigger_policy_webhook")
+def test_create_and_process_access_request_bigquery_enterprise(
+    trigger_webhook_mock,
+    bigquery_enterprise_test_dataset_config,
+    db,
+    cache,
+    policy,
+    dsr_version,
+    request,
+    policy_pre_execution_webhooks,
+    policy_post_execution_webhooks,
+    run_privacy_request_task,
+):
+    request.getfixturevalue(dsr_version)  # REQUIRED to test both DSR 3.0 and 2.0
+
+    customer_email = "customer-1@example.com"
+    user_id = (
+        1754  # this is a real (not generated) user id in the Stackoverflow dataset
+    )
+    data = {
+        "requested_at": "2024-08-30T16:09:37.359Z",
+        "policy_key": policy.key,
+        "identity": {
+            "email": customer_email,
+            "stackoverflow_user_id": {
+                "label": "Stackoverflow User Id",
+                "value": user_id,
+            },
+        },
+    }
+
+    pr = get_privacy_request_results(
+        db,
+        policy,
+        run_privacy_request_task,
+        data,
+        PRIVACY_REQUEST_TASK_TIMEOUT_EXTERNAL,
+    )
+
+    results = pr.get_raw_access_results()
+    assert len(results.keys()) == 4
+
+    for key in results.keys():
+        assert results[key] is not None
+        assert results[key] != {}
+
+    users = results["enterprise_dsr_testing:users"]
+    assert len(users) == 1
+    user_details = users[0]
+    assert user_details["id"] == user_id
+    assert user_details["display_name"] == "David R. Longnecker"
+    assert user_details["location"] == "Kansas City, MO, USA"
+    assert (
+        user_details["profile_image_url"]
+        == "https://i.stack.imgur.com/egFxf.jpg?s=128&g=1"
+    )
```

Comment:
I think the user ID is sufficient for this test, so we don't bring any unwanted attention to our user
```suggestion
```